### PR TITLE
:gift: Add tea.xyz package manager :tea: 

### PR DESCRIPTION
**Pull request by adriens**
- https://github.com/teaxyz/pantry/pull/834

### CODE REVIEW

File: docs/content/installing.md [hunk 1]
```diff
@@ -135,6 +135,19 @@ $ npm install -g gomplate
...
+[`tea.xyz`](https://tea.xyz/) maybe be very comfortable, therefore, to install, just:
+
+```console
+$ sh <(curl https://tea.xyz) +gomplate.ca sh
+$ gomplate --version
+...
+```
+
+
+
```
Comment (hairyhenderson) [inline docs/content/installing.md:150]:
```suggestion

```

**Review by hairyhenderson — APPROVED** [review]
Thanks @adriens!

**Comment by hairyhenderson** [comment]
Interesting! Thanks @adriens - this looks reasonable to add 👍

**Comment by adriens** [comment]
[:gift_heart: ☕ Make software packaging... a cup of tea.xyz ](https://dev.to/adriens/make-software-packaging-a-cup-of-teaxyz-3h77) :pray:

**Pull request merged**